INSTALL.rst.txt

```diff
@@ -14,24 +14,17 @@ Prerequisites
 
 Building NumPy requires the following installed software:
 
-1) For Python 3, Python__ 3.5.x or newer.
+1) Python__ 3.6.x or newer.
 
-   On Debian and derivative (Ubuntu): python python-dev
+   Please note that the Python development headers also need to be installed,
+   e.g., on Debian/Ubuntu one needs to install both `python3` and
+   `python3-dev`. On Windows and macOS this is normally not an issue.
 
-   On Windows: the official python installer on Python__ is enough
-
-   Make sure that the Python package distutils is installed before
-   continuing. For example, in Debian GNU/Linux, distutils is included
-   in the python-dev package.
-
-   Python must also be compiled with the zlib module enabled.
-
-2) Cython >= 0.29.2 (for development versions of numpy, not for released
-                     versions)
+2) Cython >= 0.29.13
```

Comment:
Hmm, `0.29.14` is the latest and has a Python 3.8 fix.

Reply:
`pyproject.toml` says 0.29.13 in master, do we need to bump it?

Reply:
Yes, we should do that. I'll make a pr.

Reply:
See #15160. I also updated the version here.